en_US/home/src/main/resources/rdf/i18n/en_US/tbox/firsttime/fileUploadAnnotations_en_US.n3

```diff
@@ -0,0 +1,4 @@
+@prefix vitro-public:  <http://vitro.mannlib.cornell.edu/ns/vitro/public#> .
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+
+vitro-public:storedFile rdfs:label "Stored file" .
```

Comment:
Should this have a @en-US tag? 

Also, I believe these property labels are historically all lowercase for some reason.

Reply:
Sure